### PR TITLE
Translated Descriptions

### PR DESCRIPTION
**Pull request by DahliaSR**
#1

### CODE REVIEW

File: translations/de/info/task_description.html [hunk 1]
```diff
@@ -24,101 +22,100 @@
...
         </p>
 
         <p>
-            --and it really WAS a kitten, after all.
+            – und es war zu guter Letzt wirklich ein Kätzchen
         </p>
         <p style="float: right">
-            "Through the Looking-Glass." Lewis Carroll
+            "Alice hinter den Spiegeln" von Lewis Carroll<br>
+            Aus dem Englischen von Jörg Karau, 2008
```
Comment (GiForce) [inline translations/de/info/task_description.html:69]: that's great...**starts a google-search**  congrats for discovering joergkarau-texte.de! 👾

File: translations/de/info/task_description.html [hunk 2]
```diff
@@ -24,101 +22,100 @@
...
+    HORT&nbsp;&rArr; HORN&nbsp;&rArr; <strong>KORN</strong>
 </p>
 
 <p>
-    The Robots like using digits more than letters, so we’ve changed the rules a little.
-    You are given the list of numbers with exactly the same length and you must find the shortest chain of numbers
-    to link the first number to the last like you would with the words.
+    Die Roboter bevorzugen Ziffern anstelle von Buchstaben, also haben wir die Regeln ein
+    wenig verändert.<br>
+    Gegeben wird eine Liste von Zahlen gleicher Länge. Die erste und letzte Zahl bilden
```
Comment (GiForce) [inline translations/de/info/task_description.html]: Gegeben ist...

File: translations/de/info/task_description.html [hunk 3]
```diff
@@ -24,101 +22,100 @@
...
+    den Start- und Endpunkt der Kette. Deine Aufgabe besteht darin, aus den übrigen Zahlen
+    der Liste die kürzeste, gültige Kette zu bilden.
```
Comment (GiForce) [inline translations/de/info/task_description.html]: _die kürzest gültige Kette_

File: translations/de/info/task_description.html [hunk 4]
```diff
@@ -24,101 +22,100 @@
...
 </p>
 
 <p>
-    For Example. There is a list [123, 991, 323, 321, 329, 121, 921, 125, 999].
-    The shortest way from the first to the last is:
+    <strong>Ein Beispiel:</strong><br>
+    Die Liste: [123, 991, 323, 321, 329, 121, 921, 125, 999]<br>
+    Die kürzeste, gültige Kette von der ersten (123) bis zur letzten Zahl (999) ist:
```
Comment (GiForce) [inline translations/de/info/task_description.html]: I still think _die kürzest gültige Kette_

File: translations/de/info/task_description.html [hunk 5]
```diff
@@ -24,101 +22,100 @@
...
+    und die kürzeste, gültige Kette zurückgibt.
 </p>
 
 <p>
-    <strong>Output: </strong> The shortest chain from the first to the last number as a list of integers.
+    <strong>Eingabe: </strong> Zahlen als Liste von ganzen Zahlen.
 </p>
 
+<p>
+    <strong>Ausgabe: </strong> Die kürzeste, gültige Kette von der ersten bis zur letzten Zahl als Liste.
```
Comment (GiForce) [inline translations/de/info/task_description.html:113]: déjà vu!  what's in for _die kürzest mögliche Kette_ ?

File: translations/de/info/task_description.html [hunk 6]
```diff
@@ -128,12 +125,12 @@
 </div>
 
 <p class="for_info_only">
-    <strong>How it is used: </strong>
-    This task is like pathfinding for numbers.
-    It shows how many things in this world can be represented with mathematics, even words.
+    <strong>Wozu das gut ist:</strong><br>
+    Diese Aufgabe ist eine Variante des <a href="https://de.wikipedia.org/wiki/Pathfinding">Pathfinding<a> für Zahlen.
```
Comment (GiForce) [inline translations/de/info/task_description.html]: Although it is interesting, I still don't see the optimizing part in this mission. The list given already has a given length? At least in the example.  You can't shorten the list, can you? I'm confused. Nevertheless I like that linked wikipedia here, even it goes much further than this mission. I'd prefer the original's point, which links the representation of numbers and math to words. And that what's happening in this mission so far I get it. But maybe I just don't ¯\_(ツ)_/¯

**Review by GiForce — CHANGES_REQUESTED** [review]
Maybe you'll notice, "die kürzeste, gültige Kette" should be optimized. Please.

File: translations/de/info/task_description.html (hunk 1 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html:69]: Took me a while!

File: translations/de/info/task_description.html (hunk 2 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html]: :white_check_mark:

File: translations/de/info/task_description.html (hunk 6 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html]: Not all the numbers in the given list are part of the correct chain: 
**Input:** [123, 991, 323, 321, 329, 121, 921, 125, 999] 
**Result:** [123, 121, 921, 991, 999]  

Actually, there are no words which are represented as numbers in this task, it only replaced words by numbers. But these numbers are not representing words... So I removed that sentence as it seems to be a bit confusing.

File: translations/de/info/task_description.html (hunk 3 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html]: :white_check_mark:

File: translations/de/info/task_description.html (hunk 4 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html]: :white_check_mark:

File: translations/de/info/task_description.html (hunk 5 above)
Comment (DahliaSR) [reply translations/de/info/task_description.html:113]: :white_check_mark:

**Comment by GiForce** [comment]
😳 really, didn’t see that, maybe I was comparing the german list with the english one when validating my argument. I should start to look at the files when formatted. There is no way github interprets HTML directly? 

> Not all the numbers in the given list are part of the correct chain: 
> Input: [123, 991, 323, 321, 329, 121, 921, 125, 999] 
> Result: [123, 121, 921, 991, 999]

**Comment by DahliaSR** [comment]
Nope there's not, thats why I'm doing it in a local repo on my computrer.

> really, didn’t see that, maybe I was comparing the german list with the english one when validating my argument. I should start to look at the files when formatted. There is no way github interprets HTML directly?

**Review by GiForce — APPROVED** [review]
✅

**Pull request merged**